composer.json

```diff
@@ -18,8 +18,8 @@
         "symfony/polyfill-ctype": "^1.8"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.6",
-        "sebastian/version": "^1.0.1"
+        "phpunit/phpunit": "^8.1.6",
```

Comment:
Likely a problem, since the package still supports PHP 5.x.

A bump in a minor release is highly endorsed, but that's up to the maintainer.

I can also use a different installation method if we don't want to use `"require-dev"`

Reply:
I'd prefer to keep allowing 5.3 for this library, though looking at it on the PHP website, it has been end of life for [almost 5 years](https://www.php.net/eol.php).

Personally i'd like to use another installation method, so we don't have to bump the mimimum version to `7.1`.

What do you think @Nyholm ?

Reply:
Tbh, anything < 7.2 is dead and defunct, especially when looking at newer releases. That said, modernisation of the library is out of scope in this particular patch: I'd be fine with `composer require vimeo/psalm` in a temporary path as part of the `.travis.yml` setup.